alvr/server_core/src/lib.rs

```diff
@@ -243,6 +245,57 @@ impl ServerCoreContext {
         }));
     }
 
+    pub fn get_device_motion(
+        &self,
+        device_id: u64,
+        sample_timestamp: Duration,
+    ) -> Option<DeviceMotion> {
+        dbg_server_core!("get_device_motion");
+
+        self.connection_context
+            .tracking_manager
+            .read()
+            .get_device_motion(device_id, sample_timestamp)
+    }
+
+    pub fn get_hand_skeleton(
+        &self,
+        hand_type: HandType,
+        timestamp: Duration,
+    ) -> Option<[Pose; 26]> {
+        dbg_server_core!("get_hand_skeleton");
+
+        self.connection_context
+            .tracking_manager
+            .read()
+            .get_hand_skeleton(hand_type, timestamp)
+            .copied()
+    }
+
+    pub fn get_motion_to_photon_latency(&self) -> Duration {
+        dbg_server_core!("get_total_pipeline_latency");
+
+        // self.connection_context
+        //     .statistics_manager
+        //     .read()
+        //     .as_ref()
+        //     .map(|stats| stats.motion_to_photon_latency_average())
+        //     .unwrap_or_default()
+
+        Duration::from_millis(0)
```

Comment:
The reason I'm returning 0 now is because we haven't implemented server-side prediction yet, so the samples already use predicted timestamps. (also this function is not even used yet, but it will in a few commits)

Reply:
fair enough, should maybe add a comment about it tho (not really important tho)